database/db.init

```diff
@@ -0,0 +1,129 @@
+drop schema public cascade;
+create schema public;
+--DROP TABLE public."user";
+
+CREATE TABLE public."user"
+(
+    id       serial,
+    name     character varying(250),
+    email    character varying(250),
+    password character varying(250),
+    CONSTRAINT id PRIMARY KEY (id)
+) WITH (
+      OIDS = FALSE
+    );
+
+CREATE TABLE SUBJECT
+(
+    id          SERIAL PRIMARY KEY UNIQUE,
+    name        VARCHAR,
+    description VARCHAR,
+    user_id     SERIAL REFERENCES public."user" (id)
+);
+
+CREATE TABLE EXAM
+(
+    id          SERIAL PRIMARY KEY UNIQUE,
+    name        VARCHAR,
+    description VARCHAR,
+    subject_id  SERIAL REFERENCES SUBJECT (id),
+    pass_score  VARCHAR,
+    file_name   VARCHAR,
+    date        DATE,
+    user_id     SERIAL REFERENCES public."user" (id),
+    owners      VARCHAR
+);
+
+CREATE TABLE EXAM_OWNERS
+(
+    exam_id SERIAL REFERENCES EXAM (id),
+    user_id SERIAL REFERENCES public."user" (id)
+);
+
+CREATE TABLE EXAM_FILES
+(
+    id            SERIAL PRIMARY KEY UNIQUE,
+    file_name     VARCHAR,
+    file_path     VARCHAR,
+    students_work bool,
+    answers       bool,
+    exam_id       SERIAL REFERENCES EXAM (id)
+);
+
+CREATE TABLE EXAM_RESULT_FILES
+(
+    id            SERIAL PRIMARY KEY UNIQUE,
+    file_name     VARCHAR,
+    file_path     VARCHAR,
+    exam_id       SERIAL REFERENCES EXAM (id)
+);
+
+CREATE TABLE EXAM_RESULT
+(
+    id        SERIAL PRIMARY KEY UNIQUE,
+    name      VARCHAR,
+    max_score VARCHAR,
+    score     VARCHAR,
+    student   VARCHAR,
+    exam_id   SERIAL REFERENCES EXAM (id)
+);
+
+CREATE TABLE EXAM_CORRECT_ANSWERS
+(
+    id                SERIAL PRIMARY KEY UNIQUE,
+    exam_group        VARCHAR,
+    answer_number     NUMERIC,
+    number_of_answers NUMERIC,
+    correct_answer    VARCHAR,
+    exam_id           SERIAL REFERENCES EXAM (id)
+);
+
+
+INSERT INTO public."user" (name, email, password)
+VALUES ('x', 'x@x.pl', 'c4ca4238a0b923820dcc509a6f75849b'),
+       ('admin', 'admin', 'c4ca4238a0b923820dcc509a6f75849b');
+
+
+INSERT INTO SUBJECT (name, description, user_id)
+VALUES ('MBI-2022', 'Metody badawcze w informatyce', 1),
+       ('SOWW-2022', 'Systemy o wysokiej wydajności', 1),
+       ('BS-2022', 'Bezpieczeństwo systemów', 1),
+       ('BO-2022', 'Badania operacyjne', 1),
+       ('BS-2022', 'Biblioteki cyfrowe', 1),
+       ('UO-2022', 'Użyteczność oprogramowania', 1);
+
+INSERT INTO EXAM (name, description, subject_id, user_id, pass_score, date)

```

Comment:
Shouldn't we insert `owners` into `exam` to somehow link user to exam?

Reply:
@MateuszKepczynski please have a look